content/en/infrastructure/resource_catalog/_index.md

```diff
@@ -30,70 +30,101 @@ Resource Catalog is not available in this site.
 
 ## Overview
 
-Datadog Resource Catalog provides a high-level overview of the hosts and resources in your cloud and hybrid environments. View information such as tags, configuration details, relationships between assets, misconfigurations, and threats. See what team is responsible for each resource, and what security misconfigurations have been reported. Access dashboards and Datadog views that receive and monitor telemetry and security data for each resource.
+Datadog Resource Catalog is the central hub of all your infrastructure resources to help you manage resource compliance, investigate root causes for incidents, and close observability gaps on your infrastructure. With the Resource Catalog, you can understand key resource information such as metadata, ownership, configurations, relationship between assets, and active security risks for your resources.
 
-Resource Catalog leverages Datadog cloud integrations and the Datadog Agent to gather data from cloud resources such as databases, storage services, and hosts.
+Resource Catalog leverages Datadog cloud integrations and the Datadog Agent to gather data from cloud resources such as hosts, databases, and storage services.
 
-{{< img src="/infrastructure/resource_catalog/resource_catalog_2.png" alt="The Resource Catalog page showing the Inventory tab, sorting by service" width="100%">}}
+{{< img src="/infrastructure/resource_catalog/resource-catalog-doc-img-1.png" alt="The Resource Catalog page showing the Inventory tab, sorting by service" width="100%">}}
 
-## Setup
+### Use Cases
+
+#### Governance and Reporting
+- Gain visibility into the compliance of your infrastructure with regards to ownership, versioning, migrations, etc
+- Facilitate good tagging practices to optimize cross-telemetry insights
+- Reduce application risks by identifying and fixing security vulnerabilities in the dependencies of your services
+- Provide engineering leadership with a high-level view of security practices across teams and cloud accounts.
+- Export resources for record-keeping or auditing
+
+#### Troubleshooting
```

Comment:
The reader may find this confusing since "Troubleshooting" is often a header we use for actual troubleshooting sections offering help with setup issues. Maybe something like "Troubleshooting incidents and performance issues"?

I also recommend more action-oriented headers, like "Troubleshoot incidents and performance issues" and "Optimize observability".